File: 프로그래머스/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    s1 = s.lower()
    pcount = 0
    ycount = 0

    for i in s1:
        if i == 'p':
            pcount += 1
        elif i == 'y':
            ycount += 1

    if pcount == ycount:
        answer = True
    else:
        answer = False

    return answer

File: 프로그래머스/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pcount = 0;
        int ycount = 0;
        char[] s1 = s.toCharArray();

        for (int i = 0; i < s1.length; i++) {
            if (s1[i] == 'p' || s1[i] == 'P') {
                pcount++;
            } else if (s1[i] == 'y' || s1[i] == 'Y') {
                ycount++;
            }
        }

        if (pcount == ycount) {
            answer = true;
        } else {
            answer = false;
        }

        return answer;
    }
}
